docs/cli/edgedb_connopts.rst

```diff
@@ -44,18 +44,27 @@ Connection flags
 
 :cli:synopsis:`-P <port>, --port=<port>`
     Specifies the TCP port on which the server is listening for connections.
-    Defaults to the value of the ``EDGEDB_PORT`` environment variable or, 
+    Defaults to the value of the ``EDGEDB_PORT`` environment variable or,
     if not set, to ``5656``.
 
+:cli:synopsis:`--unix-path /path/to/socket`
+    Specifies a path to a Unix socket for an EdgeDB connection. If the path is
+    a directory, the actual path will be computed using the ``port`` and
+    ``admin`` parameters.
+
+:cli:synopsis:`--admin`
+    Connect to a password-less Unix socket (specified by the ``unix_path`` with
+    superuser privileges by default.
```

Comment:
Missing an ending paren? Or maybe no parens?

```suggestion
    Connect to a password-less Unix socket. By default, specified by the
    ``unix_path`` with superuser privileges.
```

Reply:
Good catch! I think I like the parenthetical here so I went ahead and closed it.